index.src.html

```diff
@@ -1301,86 +1301,83 @@ <h4 id="monkey-patching">Monkey Patches to Fetch</h4>
   pull request against Fetch.
 
   <h4 id="body-extraction" algorithm>
-    Extract a |body| and |Content-Type| for |request|'s attached credential
+    Extract an |action| and |Content-Type| for |request|'s attached credential
   </h4>
 
-  Given a {{Request}} (|request|), this algorithm returns a {{Body}} (|body|) and
-  a string (|Content-Type|):
+  The following algorithm specializes the body mixin method to [=extract=] a
+  request body from an object, for objects of type {{PasswordCredential}}.
+  Given a {{Request}} |request| and its <a>attached credential</a>
+  |credential|, this algorithm returns an |action| and a string
+  |Content-Type|:
 
-  1.  Let |credential| be |request|'s <a>attached credential</a>.
+  1.  Let |action| be `null`, and |Content-Type| be the empty string.
```

Comment:
The default `|action|` should probably be something like "an action that returns an empty string". Returning `null` seems like it would be a compile error if we could compile specs. :)